definitions/npm/moment_v2.3.x/test_moment-v2.js

```diff
@@ -1,25 +1,37 @@
 // @flow
-import moment from "moment";
+import moment from 'moment';
 
 // Parse
-const m3: moment = moment([123, 123]);
+let m: moment;
```

Comment:
`m` is defined below, pick a different name like `m0`.

Reply:
Or use the new `describe` and `it` blocks! :)